webrender/src/platform/macos/font.rs

```diff
@@ -172,12 +211,18 @@ impl FontContext {
 
                 let rasterized_area = (metrics.rasterized_width * metrics.rasterized_height) as usize;
                 let mut rasterized_pixels = cg_context.data().to_vec();
-                for i in 0..rasterized_area {
-                    let alpha = (rasterized_pixels[i * 4 + 3] as f32) / 255.0;
-                    for j in 0..3 {
-                        rasterized_pixels[i * 4 + j] =
-                            ((rasterized_pixels[i * 4 + j] as f32) / alpha) as u8;
+
+                match render_mode {
+                    FontRenderMode::Alpha | FontRenderMode::Mono => {
+                        for i in 0..rasterized_area {
+                            let alpha = (rasterized_pixels[i * 4 + 3] as f32) / 255.0;
+                            for j in 0..3 {
+                                rasterized_pixels[i * 4 + j] =
+                                    ((rasterized_pixels[i * 4 + j] as f32) / alpha) as u8;
+                            }
```

Comment:
Could just be:

    for channel in &mut rasterized_pixels[(i*4)..(i*4 + 3)] {
        *channel = ((*channel as f32) / alpha) as u8
    }